.gitlab-ci.yml

```diff
@@ -22,8 +22,9 @@ onboarding_tests_installer:
         SCENARIO: [ SIMPLE_INSTALLER_AUTO_INJECTION, SIMPLE_AUTO_INJECTION_PROFILING ]
 
 onboarding_tests_k8s_injection:
-  variables:
-    WEBLOG_VARIANT: sample-app
+  parallel:
```

Comment:
why? isn't this a single execution? Don't we need more values in `WEBLOG_VARIANT` to take advantage of this `parallel`?

Reply:
yes, it's a single execution but it's the first PR. I'm preparing a PRs to change how k8s lib injection tests are working.
1. system-tests PR changing the K8s lib injection tests: https://github.com/DataDog/system-tests/pull/3689
2. one pipeline PR: https://github.com/DataDog/libdatadog-build/pull/59/files
3. Final PR on dd-trace-js: https://github.com/DataDog/dd-trace-js/pull/5024/files

The current PR it's working: https://gitlab.ddbuild.io/DataDog/apm-reliability/dd-trace-js/-/pipelines/51454716